pyfixdoc.py

```diff
@@ -484,7 +486,7 @@ def format_functions (
 
         for func_name, func_obj in inspect.getmembers(self.module_obj, inspect.isfunction):
             if not func_name.startswith("_"):
-                line_num, obj_md = self.document_method([self.module_name], func_name, func_obj, "function")
+                _, obj_md = self.document_method([self.module_name], func_name, func_obj, "function")
```

Comment:
meaningful variable names are important for readability.
I understand you want to resolve the `Unused variable 'line_num'` linting error, but there is a way to have both, check this out
```suggestion
                _line_num, obj_md = self.document_method([self.module_name], func_name, func_obj, "function")
```

worth noting that all of those are conventions.
Best reference on the topic in my opinion https://dbader.org/blog/meaning-of-underscores-in-python

Reply:
Personally I tend to simply replace them with `_`, with the reasoning that if they're not being used, their meaning doesn't need to be explained, as it will only clutter the lines with additional information.
However, I do recognise that having this additional information could be useful, so I'll happily go with your solution.